pkg/webhook/clusterpropagationpolicy/validating.go

```diff
@@ -46,6 +46,9 @@ func (v *ValidatingAdmission) Handle(_ context.Context, req admission.Request) a
 			return admission.Denied(err.Error())
 		}
 	}
+	if req.Operation == admissionv1.Create && policy.Spec.Association {
```

Comment:
What if the operation is `Update`? 

I suppose we can add the validation into `ValidatePropagationSpec()`. Sorry, I didn't point that out during the last review.

Reply:
> It will bother users who have already used Association, if you use 100 pp that use Association, then you need to modify these pp.

Good point! Just realized that this kind of a breaking change.

cc @chaunceyjiang for comments, do we really need to validate the never-used/deprecated field?

Reply:
I prefer to only verify the unused field of association in the create request, so that the user knows what he should use. At the same time, it reduces the interruption to existing users.

I used to cost some time on this field, and finally found the answer in github PR. “ohh,  it's not used field, not just deprecated”

During my K8S journey, the prompt is deprecated but the field should continue to work, unless the user is explicitly told that the field cannot be used.

Reply:
Can we have more clear comments on the field? Given the field was never been used, I guess nobody would expect it will work.

Reply:
Make sense, just clean the comment of field. I will create the pr to work for it. close this pr first.